control/hera/jfv3_cyclone_genesis_00.ecf_hera

```diff
@@ -16,44 +16,24 @@
 ##%include <head.h>
 ##%include <envir-p2.h>
 
-export moduleInit=/apps/lmod/lmod/init/ksh
-. $moduleInit
-
-module use /scratch1/NCEPDEV/nems/role.epic/hpc-stack/libs/intel-2022.1.2/modulefiles/stack
-module load hpc/1.2.0
-module load hpc-intel/2022.1.2
-module load hpc-impi/2022.1.2
-
-module load w3emc/2.9.2
-module load w3nco/2.4.1
-module load bacio/2.4.1
-module load sp/2.3.3
-module load sigio/2.3.2
-module load g2/3.4.5
-
-module load jasper/2.0.25
-module load libpng/1.6.37
-module load zlib/1.2.11
-module load hdf5/1.10.6
-module load netcdf/4.7.4
-module load pnetcdf/1.10.0
-
-module load wgrib2/2.0.8
-module load prod_util/1.2.2
-module load grib_util/1.2.4
+module reset
+export target=hera
+module use /../../modulefiles
+module load ${target}.lua
```

Comment:
I learned something new here.  I didn't realize you could specify that the file was .lua in a module load.